arb_os/gasAccounting.mini

```diff
@@ -680,6 +732,10 @@ public view write throw func gasAccounting_resumeTxCharges(allPurposeGas: uint,
             let osGasUsed = ((~0) - getGas()) / gasAccountingInfo.pricingParams.arbGasDivisor;
 
             set gasAccountingInfo.congestionState.gasPool = safeSubInt(gasAccountingInfo.congestionState.gasPool, int(osGasUsed));
+            set gasAccountingInfo.congestionState.gasUsageRateEstimator = rateEstimator_addUsage(
+                gasAccountingInfo.congestionState.gasUsageRateEstimator,
+                osGasUsed,
+            );
```

Comment:
Maybe safer to combine these into a single function if they're always done in combination?